restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,56 @@
 import argparse
 import json
+import locale
 import sys
 
 from .lint import lint_file
 
 
-def _main(filepath, format='text', stream=sys.stdout, encoding=None):
-    # Read and lint the file
-    errors = lint_file(filepath, encoding=encoding)
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+    # Read and lint the files
+    errors = []
```

Comment:
Still `errors` is not memory optimal. I would need to resign from `json.dumps` in favour of printing json partials on my own or to find some json iterative library that could do the trick.